src/Analyzers/Core/CodeFixes/PredefinedCodeFixProviderNames.cs

```diff
@@ -81,5 +81,6 @@ internal static class PredefinedCodeFixProviderNames
         public const string MakeStructFieldsWritable = nameof(MakeStructFieldsWritable);
         public const string AddExplicitCast = nameof(AddExplicitCast);
         public const string RemoveIn = nameof(RemoveIn);
+        public const string SimplifyLinqExpressions = nameof(SimplifyLinqExpressions);
```

Comment:
this name doesn't match the names used elsewhere (it is not pluralized wlsewhere).